src/matrixrtc/MatrixRTCSessonManager.ts

```diff
@@ -0,0 +1,130 @@
+/*
```

Comment:
There seems to be a typo: `MatrixRTCSessonManager` -> `MatrixRTCSessionManager`.

Reply:
Hah, thank you - well spotted.